src/components/Layout.jsx

```
@@ -0,0 +1,16 @@
+import Footer from "./Footer";
+import Navbar from "./NavBar";
+
+const Layout = ({ children }) => {
```

ESLint

[error] 'children' is missing in props validation

(react/prop-types)